docs/src/app/components/pages/components/Dialog/ExampleAlert.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+
+/**
+ * Alerts are urgent interruptions, requiring acknowledgement, that inform the user about a situation.
+ */
+export default class DialogExampleAlert extends React.Component {
+  state = {
+    open: false,
+  };
+
+  handleOpen = () => {
+    this.setState({open: true});
+  };
+
+  handleClose = () => {
+    this.setState({open: false});
+  };
+
+  render() {
+    const actions = [
+      <FlatButton
+        label="Cancel"
+        primary={true}
+        onTouchTap={this.handleClose}
+      />,
+      <FlatButton
+        label="Discard"
+        primary={true}
+        onTouchTap={this.handleClose}
+      />,
+    ];
+
+    return (
+      <div>
+        <RaisedButton label="Alert" onTouchTap={this.handleOpen} />
+        <Dialog
+          actions={actions}
+          modal={false}
```

Comment:
Good question. I have just tested the `Calendar` google app on my Android phone.
It's behaving like `modal={false}`.
However, Google isn't following the Material spec at 100% on all his applications. That behavior could be wrong.